Framework/include/MakeStopHemispheres.h

```diff
@@ -3,14 +3,15 @@
 
 #include "Framework/Framework/include/Hemispheres.h"
 #include "TopTagger/TopTagger/interface/TopTaggerUtilities.h"
+#include "TopTagger/TopTagger/interface/lester_mt2_bisect.h"
```

Comment:
Does it make more sense to not include this file from the TopTagger but just have our own version of it in Framework? It isn't used by the TopTagger code right?

Reply:
Apparently it gets included in `TopTaggerUtilities`, which _also_ looks like a useless include as I see no reference to any functions from that file. Also looks like `TopTaggerUtilities` is not needed in that header. Cleaning up more unnecessary includes.

Reply:
Nevermind, it is indeed used in the `TopTaggerUtilities` file.

Reply:
Alright, my fix to the `TopTagger` repo is merged and so the problem here has been resolved.